test/e2e/scalability/density.go

```diff
@@ -819,7 +821,14 @@ func createRunningPodFromRC(wg *sync.WaitGroup, c clientset.Interface, name, ns,
 			},
 		},
 	}
-	_, err := c.Core().ReplicationControllers(ns).Create(rc)
+	var err error
+	for attempt := 0; attempt < MaxLatencyPodCreationTries; attempt++ {
+		_, err = c.Core().ReplicationControllers(ns).Create(rc)
+		if err == nil {
+			break
+		}
+		Expect(!apierrs.IsAlreadyExists(err) && framework.IsRetryableAPIError(err)).To(Equal(true))
```

Comment:
I think that if it is an 'AlreadyExists' you can just proceed.

Reply:
I'm not completely sure about it - but changed it :)